lib/actions/racking.rb

```diff
@@ -31,6 +32,9 @@ def rack_layout_creating_tubes(asset_group)
       csv_parsing(asset_group, Parsers::CsvLayoutWithTubeCreation)
     end
 
+    def rack_layout_any_barcode(asset_group)
+      csv_parsing(asset_group, Parsers::CsvLayoutAnyBarcode)
```

Comment:
Passing the class type into this function looks redundant, probably clearer to pass object instance directly.